Resources/Prototypes/Recipes/Construction/structures.yml

```diff
@@ -362,6 +362,44 @@
   objectType: Structure
   placementMode: SnapgridCenter
 
+- type: construction
+  name: uranium window
+  id: UraniumWindow
+  graph: Window
+  startNode: start
+  targetNode: uraniumWindow
+  category: construction-category-structures
+  canBuildInImpassable: true
+  description: Clear and very tougher, with a RadAbsorb.
```

Comment:
Clear and much tougher than regular glass, with added RadAbsorb (patent pending) to protect you from deadly radiation.